### ISSUE TITLE
Graph not working if build task is not jnxplus

### ISSUE THREAD

**Issue by jbadeau** [khalilou88/jnxplus#120]
Hello,

I had to renamed the maven build task to:

```json
    "build": {
      "executor": "nx:noop",
      "dependsOn": ["build-maven, build-react"],
    },

    "build-maven": {
      "executor": "@jnxplus/nx-boot-maven:build",
      "dependsOn": ["^build"],
      "options": {
        "mvnArgs": "-Drevision=$version"
      },
      "outputs": ["{projectRoot}/target"]
    },

```

The nx graph does not show dependency links between apps and libs anymore.

Would be great to fix this so the plugin can live within a polyglot repo.

Regards,
Jose

**Comment by khalilou88** [comment]
Hi @jbadeau your issue is because of this part of code:
```
function isManagedProject(projectGraphNode: ProjectGraphProjectNode): boolean {
  return (
    (projectGraphNode.type === 'app' || projectGraphNode.type === 'lib') &&
    projectGraphNode.data?.targets?.build?.executor?.includes(
      '@jnxplus/nx-boot-maven'
    )
  );
}
```
I need to change it, so it will work for you.

**Comment by jbadeau** [comment]
Awesome

**Comment by khalilou88** [comment]
You can see the pull request and add a comment if needed

**Comment by khalilou88** [comment]
The change is published in the version 5.2.0.

**Comment by jbadeau** [comment]
The graph now works but I'm now getting the following errors

```
$ npx nx affected --base=$NX_BASE --head=$NX_HEAD --verbose --targets=build
TypeError: Cannot read properties of undefined (reading 'childrenNamed')
    at getDependencies (/builds/techx/techx-template-java/node_modules/@jnxplus/nx-boot-maven/src/dep-graph/lookup-deps.js:38:9)
    at Object.processProjectGraph (/builds/techx/techx-template-java/node_modules/@jnxplus/nx-boot-maven/src/dep-graph/lookup-deps.js:17:30)
    at /builds/techx/techx-template-java/node_modules/nx/src/project-graph/build-project-graph.js:125:38
    at Generator.next (<anonymous>)
    at fulfilled (/builds/techx/techx-template-java/node_modules/tslib/tslib.js:164:62)
Failed to process the project graph with "index.js". This will error in the future!
 >  NX   Running target build for 3 projects and 5 tasks they depend on:
    - spring-boot-parent
    - spring-boot-service
    - spring-boot-library
```

**Comment by khalilou88** [comment]
Can you tell me if spring-boot-parent is a 'pom' project without dependencies ?

**Comment by jbadeau** [comment]
I just looked at the code and I see I need a empty dependencies block. I had accidentally removed it. Works now. Looks like a bug though.

**Comment by khalilou88** [comment]
I prefer to publish a fix for that it's better than a crash

**Comment by khalilou88** [comment]
I published a fix for graph in the version 5.2.1

**Comment by jbadeau** [comment]
Will try it now
